### PR TITLE
Fix scanners.dm

### LINKED ISSUE #6266: Health Analyzers have obvious broken spans
(WEB REPORT BY: dylanstrategie REMOTE: 158.69.55.2:7777)
Having a condition that makes you not use health analyzers properly seems to add a broken span class

Sometimes a "<span" shows up

Check the health analyzer code thoroughly to find the broken span tags

### LINKED ISSUE #6281: Health Scanner not showing things correctly
(WEB REPORT BY: xevonix REMOTE: 158.69.55.2:7777)
http://puu.sh/kUpvH/8ca0948f5c.png
My scanner isnt showing the things like broken bones or internal bleeding. Nor does it show blood levels.
This has happened two times now and Im not sure why.

### LINKED ISSUE #6282: Health Scanner not showing things correctly
(WEB REPORT BY: xevonix REMOTE: 158.69.55.2:7777)
http://puu.sh/kUpvH/8ca0948f5c.png
My scanner isnt showing the things like broken bones or internal bleeding. Nor does it show blood levels.
This has happened two times now and Im not sure why.

### PR DESCRIPTION
**Pull request by ghost**
Fixes : 
- Major pass on Health Analyzer strings. Clumsy output now looks more "real", the code behind the scenes was made better, and br tags are now used instead of /n/t
- Use << method instead of show_message. show_message was sanitized and broke health analyzer output. Fixes #6266
- mode var now verbosity var
- "This arm/leg is broken and needs to be splinted" string commented out. No reason for this to show up when any other fracture yields "Need advanced scan for location"
- human_standard is now enabled by default on atmospherics analyzers. This should make the readout actually logical (red is supposed to be bad, and blue is supposed to be good)
- Completely removed critical failure checks. TG can fuck off
- Fixed runtime operators. Come on, you know how typecasting works
- Formatting, defines, broken spans, that kind of stuff

Fixes #6281 
Fixes #6282

Other Changes : 
- Slightly resprited the T-ray scanner (screen will no longer flicker in color outside of the crosshair, and delays on sprite were increased)
- Recut healthanalyzer.ogg in a hopeful attempt to fix the damn thing (May or may have not worked, a bunch of empty audio has been removed)

### CODE REVIEW

File: code/game/objects/items/devices/scanners.dm [hunk 1]
```diff
@@ -90,168 +99,178 @@ REAGENT SCANNER
...
-		//writepanic("[__FILE__].[__LINE__] \\/proc/healthanalyze() called tick#: [world.time]")
-		if (( (M_CLUMSY in user.mutations) || user.getBrainLoss() >= 60) && prob(50))
-			user << text("<span class='warning'>You try to analyze the floor's vitals!</span>")
-			for(var/mob/O in viewers(M, null))
-				O.show_message(text("<span class='warning'>[user] has analyzed the floor's vitals!</span>"), 1)
-			user.show_message(text("<span class='notice'>Analyzing Results for The floor:\n\t Overall Status: Healthy</span>"), 1)
-			user.show_message(text("<span class='notice'>\t Damage Specifics: [0]-[0]-[0]-[0]</span>"), 1)
-			user.show_message("<span class='notice'>Key: Suffocation/Toxin/Burns/Brute</span>", 1)
-			user.show_message("<span class='notice'>Body Temperature: ???</span>", 1)
+		if(!user.dexterity_check())
```
Comment (clusterfack) [inline code/game/objects/items/devices/scanners.dm]: No, don't do this. This proc can be used in many ways this check should not be here.

File: code/game/objects/items/devices/scanners.dm [hunk 2]
```diff
@@ -90,168 +99,178 @@ REAGENT SCANNER
...
-	else if (M.getBrainLoss() >= 100 || !M.has_brain())
-		message += "\n<span class='warning'>Subject is brain-dead.</span>"
-	else if (M.getBrainLoss() >= 60)
+			message += "<br><span class='warning'><b>Warning: [D.form] Detected</b><br>Name: [D.name].<br>Type: [D.spread].<br>Stage: [D.stage]/[D.max_stages].<br>Possible Cure: [D.cure]</span>"
+	if(M.reagents && M.reagents.get_reagent_amount("inaprovaline"))
+		message += "<br><span class='notice'>Bloodstream Analysis located [M.reagents:get_reagent_amount("inaprovaline")] units of rejuvenation chemicals.</span>"
+	if(M.has_brain_worms())
+		message += "<br><span class='warning'>Strange MRI readout. Subject needs further scanning.</span>"
+	else if(M.getBrainLoss() >= 100 || !M.has_brain())
+		message += "\n<span class='warning'>No brain activity has been detected. Subject is braindead.</span>"
```
Comment (clusterfack) [inline code/game/objects/items/devices/scanners.dm]: If you're planning on replacing all \n's you missed this one.

File: code/game/objects/items/devices/scanners.dm [hunk 3]
```diff
@@ -90,168 +99,178 @@ REAGENT SCANNER
 	melt_temperature = MELTPOINT_PLASTIC
 	origin_tech = "magnets=1;biotech=1"
 	var/last_reading = null
-	var/mode = 1;
+	var/verbosity = 1
```
Comment (clusterfack) [inline code/game/objects/items/devices/scanners.dm]: Needless change tbh

Comment (dylanstrategie) [inline code/game/objects/items/devices/scanners.dm]: Clarity is always nice

Comment (PJB3005) [inline code/game/objects/items/devices/scanners.dm]: *cough comments

**Comment by Probe1** [comment]
>Hey guys you know that thing engineers barely use because it's pretty lackluster to begin with?

>What if we nerfed it to shit and added a new item that worked like it used to just in case they might accidentally want to use it

>Yeah, good content +1

the bugfixing is :+1: 

the nerf to t rays for no reason, move to the protolathe for no reason (it's a basic tool and available in vending machines), addition of a deluxe t ray that acts like t rays do now .. for no reason

That's all garbage.

**Comment by dylanstrategie** [comment]
The deluxe T-Ray has a range of 3. The regular one wasn't changed outside of being moved to Protolathes. Worst case scenario if you are a Cargo Tech and need a T-ray scanner you need to contact Mechanic or R&D to print a spare real quick

**Comment by Probe1** [comment]
image

OH I SEE 

I apologize for this mistake.  I thought you reduced their size since 1 usually means 1 and not 3x3.  

Keep it on the autolathe though because it's basic shit that comes out of a vending machine, not research oriented.

**Comment by dylanstrategie** [comment]
I'll see what I can do. I can most likely have both

There's an AUTOLATHE fab flag but clearly it's not working so fuck that

**Comment by Probe1** [comment]
why not just not do the protolathe?  having an advanced variant in the protolathe is good.  having a basic variant is not.

**Comment by dylanstrategie** [comment]
Right then, I'll pass

Give me a minute to tweak this

**Comment by PJB3005** [comment]
@dylanstrategie yeah it doesn't work, fabricator code is a complete piece of shit.

**Comment by d3athrow** [comment]
you have no idea what the fuck atomic PRs mean do you

**Comment by ihadtoregisterforthis** [comment]
### >says the same person that improved and removed skeletons in a single merge

**Pull request merged**